app/editor/src/features/admin/notifications/dashboard/utils/getStatus.ts

```diff
@@ -0,0 +1,22 @@
+import { NotificationStatusName } from 'tno-core';
+
+/**
+ * Provides a message that represents the status.
+ * @param status The report status.
+ * @returns A message that represents the status.
+ */
+export const getStatus = (status?: NotificationStatusName) => {
```

Comment:
Seeing duplicating with subscriber and editor code.  Ideally we'd move this to `tno-core` and update each app to use the shared identical method.

Reply:
This is also different, it returns a NotificationStatusName, and the report one returns a ReportStatusName.